docs/reference/rocmcc/rocmcc.md

```diff
@@ -6,30 +6,32 @@ ROCmCC is a Clang/LLVM-based compiler. It is optimized for high-performance
 computing on AMD GPUs and CPUs and supports various heterogeneous programming
 models such as HIP, OpenMP, and OpenCL.
 
-ROCmCC is made available via two packages: rocm-llvm and rocm-llvm-alt. The
-differences are shown in this table:
-
-| **Table 1. rocm-llvm vs. rocm-llvm-alt**            |                                                                                                                               |
-|:---------------------------------------------------:|:-----------------------------------------------------------------------------------------------------------------------------:|
-| **rocm-llvm**                                       | **rocm-llvm-alt**                                                                                                             |
-| Installed by default when ROCm™ itself is installed | An optional package                                                                                                           |
-| Provides an open-source compiler                    | Provides an additional closed-source compiler for users interested in additional CPU optimizations not available in rocm-llvm |
-
-For more details, follow this table:
+ROCmCC is made available via two packages: `rocm-llvm` and `rocm-llvm-alt`.
+The differences are listed in [the table below](rocm-llvm-vs-alt).
+
+:::{list-table} Differences between `rocm-llvm` and `rocm-llvm-alt`
+:header-rows: 1
+:name: rocm-llvm-vs-alt
+
+* - **rocm-llvm**
+  - **rocm-llvm-alt**
+* - Installed by default with ROCm™
```

Comment:
Let's not switch to list tables. We can discuss tomorrow but end goal is to eventually comply with GitHub Markdown

Reply:
Reverted in [ab9a035](https://github.com/RadeonOpenCompute/ROCm/pull/1984/commits/ab9a03547b07efe20e17f66c5f332a25e2e43443)